assets/webpack.config.js

```diff
@@ -0,0 +1,62 @@
+const path = require("path");
+const glob = require("glob");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const UglifyJsPlugin = require("uglifyjs-webpack-plugin");
+const OptimizeCSSAssetsPlugin = require("optimize-css-assets-webpack-plugin");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const neat = require("bourbon-neat").includePaths;
+const bourbon = require("bourbon").includePaths;
+const bourbonNeatPaths = bourbon.concat(neat);
+console.log(bourbonNeatPaths);
+
+module.exports = (env, options) => ({
+  optimization: {
+    minimizer: [
+      new UglifyJsPlugin({ cache: true, parallel: true, sourceMap: false }),
+      new OptimizeCSSAssetsPlugin({})
+    ]
+  },
+  entry: {
+    "./js/app.js": ["./js/app.js"].concat(glob.sync("./vendor/**/*.js"))
+  },
+  output: {
+    filename: "app.js",
+    path: path.resolve(__dirname, "../priv/static/js")
```

Comment:
Possible undefined variable: '__dirname' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)